### PR TITLE
jaz-quiz-game added

### PR DESCRIPTION
**Pull request by jazbassi1**
adding a fun quiz game

### CODE REVIEW

File: Quiz.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    else:
+        print("Incorrect")
+
+    ans = int(input('5. How many days are there is February ? '))
+    if ans == 28:
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+        ans = int(input('5. How many days are there is February ? '))
```
Comment (manokhina) [inline Quiz.py]: Why do you save the answer here?

File: Quiz.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        print("Incorrect")
+
+    ans = int(input('The Final Question :) 10. What is 800 + 200 + 200 - 200 ? '))
+    if ans == 1000:
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+
+    print("Thanks for playing ! You got ", score, " questions correct. ")
```
Comment (manokhina) [inline Quiz.py]: good, but there's even more elegant way to do this :)
`print(f"Thanks for playing ! You got {score} questions correct. ")`

File: Quiz.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    mark = (score/total_questions) * 100
+
+    print("Score: ",mark, '%')
+else:
+    print('Ok Goodbye :)')
```
Comment (manokhina) [inline Quiz.py]: empty line missed

File: Quiz.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        print("Incorrect")
+
+    ans = input('4. What is the most common colour of toilet paper in France ? ')
+    if ans.lower() == 'pink':
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+
+    ans = int(input('5. How many days are there is February ? '))
```
Comment (manokhina) [inline Quiz.py]: typo `in February`

**Review by manokhina — COMMENTED** [review]
Sorry for being too picky with the typo comments sometimes, that's how it happens at work! We should aim to have perfectly clean code :)

File: Quiz.py (hunk 1 above)
Comment (jazbassi1) [reply Quiz.py]: this would be a mistake of repetition from my end. commented line out completely

File: Quiz.py (hunk 2 above)
Comment (jazbassi1) [reply Quiz.py]: added

File: Quiz.py (hunk 3 above)
Comment (jazbassi1) [reply Quiz.py]: added

File: Quiz.py (hunk 4 above)
Comment (jazbassi1) [reply Quiz.py]: updated

**Pull request merged**